cmd/config/subcommand/init/init_flags.go

```diff
@@ -0,0 +1,11 @@
+package init
+
+//go:generate pflags Config --default-var DefaultConfig
```

Comment:
You need the --bind-to-default (can't remember the exact name) if you want to bind .DefaultConfig.Host to the value the user provided